glide.yaml

```diff
@@ -0,0 +1,37 @@
+package: github.com/m3db/m3nsch
```

Comment:
For sure. I was under the impression that versions being mentioned in `glide.lock` was sufficient, will put them in `glide.yaml` too.